lhotse/features.py

```diff
@@ -0,0 +1,314 @@
+"""
+This file is just a rough sketch for now.
+"""
+from concurrent.futures.process import ProcessPoolExecutor
+from dataclasses import dataclass, field, asdict
+from functools import partial
+from itertools import chain
+from math import isclose, ceil
+from pathlib import Path
+from typing import Union, List, Iterable, Dict, Optional
+from uuid import uuid4
+
+import lilcom
+import numpy as np
+import torch
+import torchaudio
+import yaml
+
+from lhotse.audio import Recording
+from lhotse.supervision import SupervisionSegment
+from lhotse.utils import Seconds, Milliseconds, Pathlike
+
+
+@dataclass
+class SpectrogramConfig:
+    dither: float = 0.0
+    window_type: str = "povey"
+    frame_length: Milliseconds = 25.0
+    frame_shift: Milliseconds = 10.0
+    remove_dc_offset: bool = True
+    round_to_power_of_two: bool = True
+    energy_floor: float = 0.0
+    min_duration: float = 0.0
+    preemphasis_coefficient: float = 0.97
+    blackman_coeff: float = 0.42
```

Comment:
blackman_coeff makes no diff with povey window.  could remove, maybe.  How many of these are the defaults?  I wonder whether it's helpful to put them all, if we don't expect them to be changed?
BTW I think we should standardize on snip_edges = False, for simplicity of num-frames formula.

Reply:
I think a few of them differ from `torchaudio` defaults, but not a lot. My own preference is to have them explicitly defined so that if a default value changes in future torchaudio release and we update, we don't get surprised.